tests/data/scripts/hello.py

```diff
@@ -0,0 +1,24 @@
+#!/usr/bin/env python
+
+import argparse
+import warnings
+
+
+def parse_args():
+    parser = argparse.ArgumentParser(description='Say hello.')
+    parser.add_argument('name', help='To whom.')
+
+    args = parser.parse_args()
+
+    return args
+
+
+def main():
+    args = parse_args()
+    print(f'hello {args.name}!')
+    if args.name == 'lizz':
```

Comment:
Is it better to use a more general name rather than using your ID? Though I am OK with that.

Reply:
haha